src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -121,7 +121,7 @@
     <value>PowerShell {0}
 Copyright (c) Microsoft Corporation. All rights reserved.
 
-https://aka.ms/pscore6-docs
+https://aka.ms/powershell-docs
```

Comment:
Why not https://aka.ms/powershell which is more high level and might be more useful to users whilst they can still get to the https://aka.ms/powershell-docs from there

Reply:
^ @bergmeister  That does seem to be a more appropriate landing page.

Reply:
@joeyaiello thoughts?

Reply:
I'm ok changing this to the top level link.